crates/ruff_linter/src/rules/flake8_executable/rules/shebang_missing_executable_file.rs

```diff
@@ -28,7 +28,11 @@ use crate::rules::flake8_executable::helpers::is_executable;
 ///
 /// A file is considered executable if it has the executable bit set (i.e., its
 /// permissions mode intersects with `0o111`). As such, _this rule is only
-/// available on Unix-like systems_, and is not enforced on Windows or WSL.
+/// available on Unix-like systems_.
+///
+/// If this rule produces a significant amount of false positives, it's likely
+/// your source is stored on a filesystem that does not implement Unix file
+/// permissions (e.g. FAT variants, NTFS, CIFS network mounts, VM/WSL shared folders, etc).
```

Comment:
I understood that it's less about where the file is stored but more about how the file was created (WSL side or from within windows). 

Should we also document how users can fix their file system state, e.g. by running chmod from within WSL and setting the file to its default permissions?